spec/examples.txt

```diff
@@ -0,0 +1,7 @@
+example_id                                             | status  | run_time        |
+------------------------------------------------------ | ------- | --------------- |
+./spec/controllers/carebooks_controller_spec.rb[1:1:1] | failed  | 0.00906 seconds |
+./spec/controllers/carebooks_controller_spec.rb[1:2:1] | passed  | 0.12554 seconds |
+./spec/controllers/carebooks_controller_spec.rb[1:2:2] | passed  | 0.12592 seconds |
+./spec/helpers/user_controller_helper_spec.rb[1:1]     | pending | 0.00001 seconds |
+./spec/helpers/user_helper_spec.rb[1:1]                | pending | 0.00002 seconds |
```

Comment:
@GarrettCooper This is just ephemera used by rspec. You should probably `git rm` it and add it to the `.gitignore` file.

Reply:
I wondered about that. Thanks for the feedback. 

I'll let Andy handle that in the master copy.